controllers/v1/observationSubmissionsController.js

```diff
@@ -1131,5 +1131,67 @@ module.exports = class ObservationSubmissions extends Abstract {
     })
   }
 
+
+  
+  /**
+  * @api {post} /assessment/api/v1/observationSubmissions/list  List Observation Submissions
+  * @apiVersion 1.0.0
+  * @apiName List Observation Submissions
+  * @apiGroup Observation Submissions
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
+  * {
+    "message": "Successfully fetched observation submissions",
+    "status": 200,
+    "result": [
+        {
+             "_id": "5d8de379bccbfb51d4450d05",
+            "entityId": "5bfe53ea1d0c350d61b78d0f",
+            "entityExternalId": "1208138",
+            "entityType": "school",
+            "status": "started",
+            "submissionNumber": 1,
+            "updatedAt": "2019-09-27T10:24:57.182Z",
+            "createdAt": "2019-09-27T10:24:57.182Z"
+        }
+    ]
+  }
+  */
+
+   /**
+   * List of Observation submissions.
+   * @method
+   * @name list
+   * @param {String} 
+   * @returns {JSON} - Observation submissions list
+   */
+
+  async list(req) {
+    return new Promise(async (resolve, reject) => {
+
+      try {
+
+        let submissionDocument = await observationSubmissionsHelper.list
```

Comment:
@deepakr123  If solutionId,observationId and entityId is not present where is the check for error handling.

Reply:
resolved